scripts/o3de/o3de/android_support.py

```diff
@@ -170,6 +170,11 @@ def register_setting(key: str, description: str, default: str= None,  is_passwor
 SETTINGS_EXTRA_CMAKE_ARGS       = register_setting(key='extra.cmake.args',
                                                    description='Optional string to set additional cmake arguments during the native project generation within the android gradle build process')
 
+# CARBONATED -- begin : currently we support a sole install-time asset pack with size up to 1.5 Gb
+CARBONATED_AAB_ASSET_PACK_NAME  = register_setting(key='carbonated.aab.asset.pack.name',
+                                                   description='Optional: the name of play asset delivery pack name in AAB file. If it is empty, the separate asset module is not created'
```

Comment:
Is there a reason to call it something specific?  Maybe this can just be a default name, like literally "game-asset-pack" or something?    Unless we need to have fine control over the naming for some reason, it'd be fine to just always use a default name, and at best have the option to override it.

Reply:
Updated. Replaced to BOOL flag.
Using hardcoded asset project name 'app_asset_pack' to consistent with base project name 'app'